### DISCUSSION TITLE
How do I initiate Undetected ChromeDriver in C#?

### DISCUSSION THREAD

**Discussion by airparo23** [ultrafunkamsterdam/undetected-chromedriver#1645] [General]
I have a project in C# and I am trying to switch to Undetected ChromeDriver but I have no idea how to initiate it after installing it in the NuGet Manager. Could you please help me. 

I added "using Selenium.WebDriver.UndetectedChromeDriver;"
and I need to edit this line as well `WebDriver webDriver = new ChromeDriver(chromeDriverService, option);`

How should it look like? Do I need to do anything else?

**Comment by huyttdn** [comment]
```
using Selenium.WebDriver.UndetectedChromeDriver;

var webBrowser = Selenium.WebDriver.UndetectedChromeDriver.UndetectedChromeDriver.Instance(false);
webBrowser.GoTo("https://google.com");
```
